core/drivers/atmel_piobu.c

```diff
@@ -64,8 +64,8 @@ static struct gpio_chip secumod_chip;
  * gpio_pin:    pin from which value needs to be read
  * Return target GPIO pin level.
  */
-static enum gpio_level gpio_get_value(struct gpio_chip *chip __unused,
-				      unsigned int gpio_pin)
+static enum gpio_level secumod_gpio_get_value(struct gpio_chip *chip __unused,
```

Comment:
What's a "secumod"?

Reply:
That is actually the name of this IP. Named piobu in the device-tree but this actually is just a subset of the secumod which embeds much more control that just IOs

Reply:
OK, thanks.